### PR TITLE
2375 Style changes to table menu and delete

### PR DESCRIPTION
**Pull request by smitfire**
Workflow step view changes made to menu and delete table feature added.
CleanShot 2021-06-08 at 10 06 00@2x
CleanShot 2021-06-08 at 10 05 56@2x

Fixes BlueBrain/nexus/issues/2375

### CODE REVIEW

File: src/shared/containers/DataTableContainer.tsx [hunk 1]
```diff
@@ -5,14 +5,28 @@ import * as React from 'react';
...
   Table,
+  Col,
+  Row,
   Button,
+  Typography,
   Input,
   Space,
   Spin,
   Modal,
+  Divider,
```
Comment (dhaneshnm) [inline src/shared/containers/DataTableContainer.tsx:16]: This looks like an unused import.

File: src/shared/containers/DataTableContainer.tsx [hunk 2]
```diff
@@ -5,14 +5,28 @@ import * as React from 'react';
 import {
   Tag,
   Table,
+  Col,
+  Row,
   Button,
+  Typography,
   Input,
   Space,
```
Comment (dhaneshnm) [inline src/shared/containers/DataTableContainer.tsx:13]: Unused?

File: src/shared/containers/DataTableContainer.tsx [hunk 3]
```diff
@@ -75,17 +92,63 @@ const DataTableContainer: React.FC<DataTableProps> = ({
...
+      projectLabel,
+      encodeURIComponent(tableResourceId),
+      latest._rev
+    );
+    console.log('latest deprecated table function');
+    // console.log(latest);
+    console.log(deprecated);
+    return deprecated;
+  };
+  const deprecateTableResource = useMutation(deprecateTable, {
```
Comment (dhaneshnm) [inline src/shared/containers/DataTableContainer.tsx:125]: Let us remove this, since we are not using it.

**Review by dhaneshnm — APPROVED** [review]

**Review by dhaneshnm — DISMISSED** [review]

**Review by Stafeeva — DISMISSED** [review]

**Review by Stafeeva — DISMISSED** [review]

**Review by dhaneshnm — APPROVED** [review]

**Review by dhaneshnm — APPROVED** [review]

**Review by Stafeeva — APPROVED** [review]
🔥

**Pull request merged**